website/blog/2018-08-27-7.0.0.md

```diff
@@ -0,0 +1,444 @@
+
+## What's Next?
+
+- Babel is inherently tied to what it compiles: JavaScript. As long as there are new additions to propose/work on there is work to be done there. That includes the time/effort to implement and maintain the syntax even before it becomes "stable". We care about the whole process: the upgrade path, education of new features, teaching of standards/language design, ease of use, and integration with other projects.
+  - Related: we are almost finished implementing [the new decorators proposal](https://github.com/babel/babel/pull/7976) in Babel thanks to [Nicolò](). It has been a long journey (it has taken [more than one year](https://github.com/babel/babel/pull/6107#issue-135801160)!) because the new proposal is completely different and much more powerful than the old one, but it is almost there :tada:. You can expect it to be released in one of the next minor versions, alongside with a blog post which will explain the changes in detail.
+- There are a lot of new features in the works: plugin ordering, better validation/errors, speed, re-thinking loose/spec options, caching, using Babel asynchronously, building against itself from CI, smoke tests, running test262. Check out this [roadmap](http://babeljs.io/docs/en/next/roadmap) doc for some more possible ideas!
+
+We have no secret plans: we're trying the best we can with what we have to serve this community.
+
+## Open Source is a Mirror
+
+I'd love if we'd have the time and resources to accomplish all of these ideas and to do it well. But as we've shown with this current release, things take much longer than expected!
+
+Why do these releases take so long anyway? Is it from the feature creep, both from ourselves and our users? Was it because we don't understand how to prioritize among all the possible things to add or fix? Deciding to fix low-hanging fruit vs. fundamental issues until the end? Or "distractions" from helping others on GitHub, Slack, Twitter? Are we just bad are estimating our time, understanding the complexities of these issues, overcommitting as volunteers?
+
+Or are we just setting too high of an expectation on ourselves, or feel so pressured by others to perform and fit to their needs at the harm of ourselves? I can only describe it as dread when seeing a message from someone online wondering why something hasn't been released while another asks why this bug isn't fixed yet. I want to just rush it out and be done with it but I also have a desire to take this seriously. 
+
+I've tried to [express](https://twitter.com/left_pad/status/1032348719931432961) some of these thoughts and struggles in my talk last week at React Rally: [Through the (Open Source) Looking Glass](https://www.youtube.com/watch?v=h0sfFX7WH1c), which I hope you can take an opportunity to listen to. The question I ask myself: What can I do about the inevitable cycle of maintainer burnout, constant anxiety, and general entitlement?
```

Comment:
"general entitlement" feels a little out of place here without some kind of explanation. Could this be replaced with something like "unrealistic expectations"? I don't know if that's the right way to phrase that,  but I'm not sure the current phrasing gets the point across in a constructive way (even though I, as a maintainer, understand what the intent is).

Reply:
I agree 👍